apt/minimization/minimizer.py

```diff
@@ -227,27 +280,27 @@ def fit(self, X=None, y=None):
                 while accuracy > self.target_accuracy:
                     nodes = self._get_nodes_level(level)
                     self._calculate_level_cells(level)
```

Comment:
This code either needs to catch the exception thrown by _calculate_level_cells, or check for itself whether level > self.dt_.get_depth() (this is actually another stop condition of the loop).

Reply:
do you want it to continue running and just break the loop or to finish the run and raise error?

Reply:
Break the loop